cs/ccxt/base/Exchange.cs

```diff
@@ -164,6 +164,12 @@ public async virtual Task<object> fetch(object url2, object method2 = null, obje
         var headers = this.extend(this.headers, headers3) as dict;
         var body = body2 as String;
 
+        var proxyUrl = this.checkProxyUrlSettings (url, method, headers, body);
+        if (proxyUrl != null) {
+            proxyUrl = proxyUrl.ToString();
```

Comment:
it shouldn't be called `Callback` callback means a different thing, more like when you call it back after an action

Reply:
yeah ofc, but the reason I used `callback` name is that we already used it for  similar functions:
https://github.com/ccxt/ccxt/blob/master/ts/src/base/Exchange.ts#L193-L197

Reply:
however, i'll rename this new one yeah

Reply:
done